src/createServer.js

```diff
@@ -1,3 +1,37 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const convertToCase = require('./convertToCase/convertToCase').convertToCase;
+const getErrors = require('./getErrors').getErrors;
+
+function createServer() {
+  const server = http.createServer((request, response) => {
+    const currentURL = new URL('htto://' + request.url);
+
+    response.setHeader('Content-Type', 'application/json');
+
+    const requestText = currentURL.pathname.slice(1);
+    const requestCase = currentURL.searchParams.get('toCase');
+    const errors = getErrors(requestText, requestCase);
+
+    if (!errors.errors.length) {
+      const {
+        convertedText,
+        originalCase,
+      } = convertToCase(requestText, requestCase);
+
+      const convertResult = {
+        originalCase,
+        targetCase: requestCase,
+        originalText: requestText,
+        convertedText,
+      };
+
+      response.end(JSON.stringify(convertResult));
+    } else {
+      response.end(JSON.stringify(errors));
+    }
```

Comment:
You forget about status and statusText 
![image](https://github.com/mate-academy/node_case-transformer-server/assets/63243136/d975100b-f478-43c2-8241-9065de79657a)
![image](https://github.com/mate-academy/node_case-transformer-server/assets/63243136/84a1ee01-a563-42ea-a9b9-fa8987dae297)

Also, avoid else statement